extension/maxminddb.c

```diff
@@ -107,9 +107,64 @@ static int Reader_init(PyObject *self, PyObject *args, PyObject *kwds)
 static PyObject *Reader_get(PyObject *self, PyObject *args)
 {
     char *ip_address = NULL;
+    PyObject *object = NULL;
+#if PY_MAJOR_VERSION >= 3
+    PyObject *bytes = NULL;
```

Comment:
No need to declare the variables at the top of the function.

Reply:
Where do you propose I declare it? I don't think I can declare it inside the "if" condition since that will cause "bytes" to be scoped to that block which will mean it can't be used in the code further down.

Reply:
I also can't declare `object` inside the "if" condition because I need to use `object` in the condition expression.